desks-web-app/src/components/booking-components/WeekOverviewRow.vue

```diff
@@ -66,17 +66,27 @@ export default defineComponent({
     previousWeek() {
       this.window--
       this.startOfWeek = this.weeks[this.window]
-
     },
     isPreviousWeekDisabled(): boolean {
       return this.window <= 0
     },
-    isDisabled: (startOfDay: Moment): boolean => startOfDay.isBefore(moment().startOf("day")),
-
-    calculateNthDayOfWeek(n: number): Moment {
-      return moment(this.startOfWeek).add(n, "day")
+    getNext5WorkingDays(start: Moment): Moment[] {
+      const days = new Array<Moment>(5);
+      days[0] = this.getNextWorkingDay(start);
+      for(let i = 0; i < 5; i++) {
+        days[i+1] = this.getNextWorkingDay(days[i]);
+      }
+      return days;
     },
 
+    getNextWorkingDay(d: Moment) {
```

Comment:
```suggestion
    getNextBusinessDay(d: Moment) {
```
Would be better English